### ISSUE TITLE
Make `publishable` required, rename some checks in assets-for-api-docs repo

### ISSUE THREAD

**Issue by gspencergoog** [flutter/flutter#88613]
## Description

In the [`flutter/assets-for-api-docs`](https://github.com/flutter/assets-for-api-docs) repo, there is a new test called `publishable` that should be made required.

I'd also like to rename `tests-linux` to `tests`, and `analyze` to `analyze+format` to be more in line with the `packages` and `plugins` repos, but I don't have permission to make those tests not be required in order to submit the change.

**Comment by CaseyHillers** [comment]
\cc @godofredoc are you able to enable this in the admin settings?

**Comment by yusufm** [comment]
Marking p3 because its not blocking.

**Comment by godofredoc** [comment]
> \cc @godofredoc are you able to enable this in the admin settings?

I believe I can, let me check

**Comment by godofredoc** [comment]
Added `publisable` and removed the other ones to allow renaming them. Once the name change has landed please reassign the bug to me.

**Comment by godofredoc** [comment]
@gspencergoog are we ready to add the new task names to the list?

**Comment by gspencergoog** [comment]
@godofredoc Not yet, but once flutter/assets-for-api-docs#169 lands, yes.

**Comment by gspencergoog** [comment]
@godofredoc OK, we're ready to add the new tasks to the list.

**Comment by godofredoc** [comment]
Done, please reopen if you find any issues.
